app/views/videos/show.html.haml

```diff
@@ -10,5 +10,36 @@
         %p= @video.description
         .actions
           %a.btn.btn-primary Watch Now
-          %a.btn.btn-default + My Queue
+          = link_to "+ My Queue", queue_items_path(video_id: @video.id), method: "POST", class: "btn btn-default"
+
+
 %section.reviews.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = form_for([@video, @review]) do |f|
+        %fieldset
+          .form-group
+            = f.label :rating, "Rate this video"
+            .row
+              .col-sm-3
+                = f.select("rating", options_for_select([5, 4, 3, 2, 1].map {|n| [pluralize(n, "Star"), n]}), {}, {class: "form-control"})
```

Comment:
I would suggest to move `options_for_select ... etc` to a helper method. I sort of clutters the view. Maybe name the helper method as `options_for_rating` and thats it.